hash_test.go

```diff
@@ -18,6 +18,23 @@ type merkleHashTest struct {
 	output, error string
 }
 
+type signatureRootTest struct {
+	val1 interface{}
+	val2 interface{}
+}
+
+type blockHeader struct {
+	Slot              uint64
+	PreviousBlockRoot []byte
+	Signature         []byte
+}
+
+type truncateTestStruct struct {
+	Slot              uint64
+	StateRoot         []byte
+	PreviousBlockRoot []byte
```

Comment:
instead of `previousBlockRoot` use a better field name, ex: `dummyField`, `ignoredField`, etc .